usage/dist/src/main/license/LICENSE

```diff
@@ -0,0 +1,1691 @@
+org.apache.jclouds.driver-jclouds-okhttp-1.9.0.jar
+org.apache.jclouds.driver-jclouds-slf4j-1.9.0.jar
+org.apache.jclouds.driver-jclouds-sshj-1.9.0.jar
+org.apache.jclouds.labs-abiquo-1.9.0.jar
+org.apache.jclouds.labs-docker-1.9.0.jar
+org.apache.jclouds.labs-google-compute-engine-1.9.0.jar
+org.apache.jclouds.labs-googlecloud-1.9.0.jar
+org.apache.jclouds.labs-oauth-1.9.0.jar
+org.apache.jclouds.provider-aws-ec2-1.9.0.jar
+org.apache.jclouds.provider-aws-s3-1.9.0.jar
+org.apache.jclouds.provider-azureblob-1.9.0.jar
+org.apache.jclouds.provider-cloudfiles-uk-1.9.0.jar
+org.apache.jclouds.provider-cloudfiles-us-1.9.0.jar
+org.apache.jclouds.provider-elastichosts-lon-b-1.9.0.jar
+org.apache.jclouds.provider-elastichosts-lon-p-1.9.0.jar
+org.apache.jclouds.provider-elastichosts-sat-p-1.9.0.jar
+org.apache.jclouds.provider-go2cloud-jhb1-1.9.0.jar
+org.apache.jclouds.provider-gogrid-1.9.0.jar
+org.apache.jclouds.provider-hpcloud-compute-1.9.0.jar
+org.apache.jclouds.provider-hpcloud-objectstorage-1.9.0.jar
+org.apache.jclouds.provider-openhosting-east1-1.9.0.jar
+org.apache.jclouds.provider-rackspace-cloudservers-uk-1.9.0.jar
+org.apache.jclouds.provider-rackspace-cloudservers-us-1.9.0.jar
+org.apache.jclouds.provider-serverlove-z1-man-1.9.0.jar
+org.apache.jclouds.provider-skalicloud-sdg-my-1.9.0.jar
+org.apache.jclouds.provider-softlayer-1.9.0.jar
+
+-----------------------------------------------------------------------------
+
+aopalliance-aopalliance-1.0.jar
+
+LICENCE: all the source code provided by AOP Alliance is Public Domain.
+http://aopalliance.sourceforge.net/
+
+-----------------------------------------------------------------------------
+
+ch.qos.logback-logback-classic-1.0.7.jar
+ch.qos.logback-logback-core-1.0.7.jar
+
+Logback: the reliable, generic, fast and flexible logging framework.
+Copyright (C) 1999-2012, QOS.ch. All rights reserved.
+
+This program and the accompanying materials are dual-licensed under
+either the terms of the Eclipse Public License v1.0 as published by
+the Eclipse Foundation
+
+  or (per the licensee's choosing)
+
+under the terms of the GNU Lesser General Public License version 2.1
+as published by the Free Software Foundation.
+
+The Eclipse Public License v1.0 is reproduced below.
+
+-----------------------------------------------------------------------------
+
+com.beust-jcommander-1.27.jar
+
+Copyright 2010 Cedric Beust cedric@beust.com
+Licensed under the Apache License v2.0
+
+-----------------------------------------------------------------------------
+
+com.fasterxml.jackson.core-jackson-annotations-2.4.0.jar
+com.fasterxml.jackson.core-jackson-core-2.4.2.jar
+com.fasterxml.jackson.core-jackson-databind-2.4.2.jar
+com.fasterxml.jackson.jaxrs-jackson-jaxrs-base-2.4.2.jar
+com.fasterxml.jackson.jaxrs-jackson-jaxrs-json-provider-2.4.2.jar
+com.fasterxml.jackson.module-jackson-module-jaxb-annotations-2.4.2.jar
+
+Licensed under the Apache License v2.0
```

Comment:
They are grouped by the organisation that created them. Some organisations include a Copyright statement (like jcommander in this example) so it is convenient to group by organisation. Not all do (like fasterxml in this example).

No strong opinion on this one; I can group by license instead of organisation if that is preferred.